python/lsst/ts/electrometer/controller.py

```diff
@@ -622,7 +630,16 @@ async def get_intensity(self):
         res = await self.send_command(f"{self.commands.get_measure(1)}", has_reply=True)
         res = res.split(",")
         res = res[0].strip("ZVDCNA")
-        self.last_value = float(res)
+        # If the range saturates the intensity negatively, the device returns
+        # +9.90000+E37O with an O not zero
+        try:
+            self.last_value = float(res)
+        except ValueError:
+            self.last_value = float("inf")
+        # If the range saturates the intensity positively, the device returns
+        # +9.90000+E37
+        if float(res) == 9.9e37:
```

Comment:
This is a magic number. Please extract to a constant and add a doc string indicating what the value means.